src/database/actions/cabinets.rs

```diff
@@ -302,6 +322,59 @@ pub async fn modify_mixer_in_cabinet(
     Ok(())
 }
 
+pub async fn modify_mixer_in_cabinet_rsm(
+    id: i32,
+    mixer_id: i32,
+    amount: Option<i32>,
+    pool: &Pool<Postgres>,
+) -> Result<(), potion::Error> {
+    sqlx::query("UPDATE cabinet_mixers SET amount = $1 WHERE cabinet_id = $2 AND id = $3")
+        .bind(amount)
+        .bind(id)
+        .bind(mixer_id)
+        .execute(pool)
+        .await
+        .map_err(|e| QueryError::from(e).into())?;
+
+    update_cabinet_checksum(id, pool).await?;
+
+    Ok(())
+}
+
+/// Sets a mixer as usable. DOES NOT CHECK OWNERSHIP OR RIGHTS
+pub async fn set_mixer_usable(
+    id: i32,
+    cabinet_id: i32,
+    pool: &Pool<Postgres>,
+) -> Result<(), potion::Error> {
+    sqlx::query("UPDATE cabinet_mixers SET usable = true WHERE id = $1 AND cabinet_id = $2")
+        .bind(id)
+        .bind(cabinet_id)
+        .execute(pool)
+        .await
+        .map_err(|e| QueryError::from(e).into())?;
+
+    update_cabinet_checksum(cabinet_id, pool).await?;
+    Ok(())
+}
+
+/// Sets a mixer as unusable. DOES NOT CHECK OWNERSHIP OR RIGHTS
```

Comment:
Nitpicking, but unnecessary comment because:
- Self explanatory function name
- Unnecessary mention of access rights, as it is default for all of the methods in file.